rllib/offline/dataset_reader.py

```diff
@@ -53,6 +54,12 @@ def get_dataset_and_shards(
         dataset = ray.data.read_parquet(
             path, parallelism=parallelism, ray_remote_args={"num_cpus": cpus_per_task}
         )
+    elif format == "native":
+        # Todo: This should probably not be named path
+        if callable(path):
```

Comment:
pretty sure not many users are using this yet.
so I propose we change the path param to loader_fn.
and by default, we would just do:

```
input_config = {
  loader_fn=dataset_loader_from_path("path")
}
```
WDYT?

Reply:
Sounds good to me, I'll up the PR

Reply:
I see, there is always a catch :)
yeah, I think type and path together can be a short cut for a generic dataset loader 👌